README.md

```diff
@@ -81,27 +106,39 @@ vercel_runtime = { version = "0.2.1" }
 name = "handler"
 path = "api/handler.rs"
 
-# Note that you need to provide unique names for each binary
-[[bin]]
-name = "user-id"
-path = "api/user/[id].rs"
-
-[[bin]]
-name = "group-id"
-path = "api/group/[id].rs"
+# Note that you need to provide unique names for each binary:
+# [[bin]]
+# name = "user-id"
+# path = "api/user/[id].rs"
+#
+# [[bin]]
+# name = "group-id"
+# path = "api/group/[id].rs"
 
 # --snip--
 ```
 
+Finally make sure to ignore the build artifacts of Rust in your `.vercelignore`.
+
+```shell
+target/
+```
+
+### Local Development
+
+You can start local development by installing [Vercel CLI](https://vercel.com/docs/cli#installing-vercel-cli) and running `vercel dev`.
+
+Please ensure `rust` and `cargo` are already installed and available in your `PATH`, since they will not be installed automatically. The recommended way to install is with [rustup](https://rustup.rs/).
+
 ### Dependencies
 
-This Builder supports installing dependencies defined in the `Cargo.toml` file.
+The builder module supports installing dependencies defined in the `Cargo.toml` file.
 
-Furthermore, more system dependencies can be installed at build time with the presence of a shell `build.sh` file in the same directory as the entrypoint file.
+Furthermore, more system dependencies can be installed at build time with the presence of a shell `build.sh` file in the root directory of your project.
 
 ## Prebuilt Deployments
 
-When creating a prebuilt deployment, the build output must be for x86_64 linux. To do this, create a Cargo build configuration at `.cargo/config.toml` with the following contents:
+When creating a prebuilt deployment, the build output must be for `x86_64 linux`. To do this, create a Cargo build configuration at `.cargo/config.toml` with the following contents:
 
 ```toml
```

Comment:
Put these under an ```# Advanced Use``` heading or something. 
So it's 

-- What the package is 
-- Getting Started 
-- Advanced use
   -- Managing dependencies 
   -- Prebuilt deployments 
   -- Function bundling 
   -- etc.